content/guides/secure-payments-by-bookingsync.md

```diff
@@ -0,0 +1,121 @@
+# Secure Payments By BookingSync
+
+1. TOC
+{:toc}
+
+## Preface
+
+If you have an integration with BookingSync and you create bookings on your side according to the instant booking flow (i.e. a booking gets confirmed only when the credit card payment is executed successfully) and you don't have PCI DSS compliant platform for collecting credit card payments, you can use our Secure Payments By BookingSync flow for this purpose.
```

Comment:
Should still require at least PCI-DSS 3.1+ SAD-A when using a full client side process (JS SDK) and SAQ-D (server-to-server integration) otherwise.